client/integration-tests/generate-flags/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+quarkus.openapi-generator.codegen.spec.generate_models_false_yaml.generate-models = false
+quarkus.openapi-generator.codegen.spec.generate_apis_false_yaml.generate-apis = false
```

Comment:
What will be generated in a scenario like this? What's the use case?

Reply:
I'm thinking about the motivation of this PR since it won't make much sense not to generate the APIs or models since one depends on the other. One can generate only the models for some weird use cases out there. Can you clarify? Do you have a use case where you would toggle any of these options?

Reply:
Ah, sorry I didn't get the question. I thought you were referring to skipping both apis and models at the same time. 

The reasoning for this is that we want to autogenerate three Java libraries for an openapi.yaml
1) models only
2) server stubs
3) client library

And we would like the server stubs and client library to share the same models library to eliminate some mapping between the client and server models that happens throughout our code.